apps/web/app/(use-page-wrapper)/[user]/page.tsx

```diff
@@ -30,7 +30,8 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
     () => profile.name,
     () => markdownStrippedBio,
     false,
-    getOrgFullOrigin(entity.orgSlug ?? null)
+    getOrgFullOrigin(entity.orgSlug ?? null),
+    `/${params.user}`
```

Comment:
<img width="490" alt="Screenshot 2025-02-14 at 8 45 29 AM" src="https://github.com/user-attachments/assets/b74e1c9d-8a89-48f6-b1d4-18f2ef564f6b" />


`og:url` is being created correctly